tests/unit/acquisition/multi_objective/test_pareto.py

```diff
@@ -104,3 +105,74 @@ def test_get_reference_point_extract_based_on_pareto_front(
     observations: tf.Tensor, expected: tf.Tensor
 ) -> None:
     tf.debugging.assert_equal(get_reference_point(observations), expected)
+
+
+@pytest.mark.qhsri
```

Comment:
what does this do? `@pytest.mark.qhsri`

maybe I'm misreading this but it looks like it's a restricted test, while this tests our generic Pareto object?